src/main/java/io/xstefank/wildfly/bot/PullRequestRuleProcessor.java

```diff
@@ -49,14 +49,14 @@ void pullRequestRuleCheck(
         for (WildFlyConfigFile.WildFlyRule rule : wildflyBotConfigFile.wildfly.rules) {
             if (Matcher.notifyRequestReview(pullRequest, rule)) {
                 if (!rule.notify.isEmpty()) {
-                    LOG.infof("Pull Request %s was matched with a rule, containing notify, with the id: %s.",
+                    LOG.infof("Pull Request \"%s\" was matched with a rule, containing notify, with the id: %s.",
```

Comment:
This is forgotten change from custom logger. I think we don't want something like "Pull Request [#123] - Pull Request "title-123"..." but rather "Pull Request [#123] - title "title-123"... "
```suggestion
                    LOG.infof("title \"%s\" - was matched with a rule, containing notify, with the id: %s.",
```
Or we can update logger with new methods or flags specifying what all do we want to print, i.e. only number / number with title etc. Having default only number for example

Reply:
go for it